jormungandr/src/network/p2p/policy.rs

```diff
@@ -40,6 +41,8 @@ pub struct PolicyConfig {
     max_quarantine: Option<Duration>,
     #[serde(default)]
     max_num_quarantine_records: Option<usize>,
+    #[serde(default)]
+    quarantine_whitelist: Option<HashSet<std::net::SocketAddr>>,
```

Comment:
use the [`Address`](https://docs.rs/poldercast/0.13.0/poldercast/struct.Address.html) directly here. It supports `serde` already. and also it supports extending the support for FQDN later on if needed.